novawallet/Modules/DApp/DAppOperationConfirm/ViewModel/DAppOperationBytesConfirmViewModelFactory.swift

```diff
@@ -0,0 +1,5 @@
+final class DAppOperationBytesConfirmViewModelFactory: DAppOperationConfirmViewModelFactory {
```

Comment:
I would make Base class and 2 implementations that override base class just two avoid cases where we are changing something for extrinsics forgetting that bytes inherit it.